_includes/what-we-do.html

```diff
@@ -35,8 +35,8 @@
         <h3>Network security</h3>
         <p>
           A secure VPC configuration with multiple environments (staging, production, mgmt), multiple layers of
-          subnets (public, private, persistence), strict security groups, NAT gateways, SSL certificates, and a bastion
-          host for secure SSH access.
+          subnets (public, private, persistence), strict security groups, NAT gateways, and a bastion host for secure
```

Comment:
I agree SSL is great to help people with, but it seemed like the only thing that wasn't part of the VPC. Though technically it is network security.  

I'm ok to leave it in, but I'll change to `TLS/SSL certificates`.